.buckconfig

```diff
@@ -7,3 +7,4 @@
 
 [alias]
   movies = //Examples/Movies/android/app:app
```

Comment:
could you add a comment how to run those apps with BUCK?
A prerequisite right now is:
```
npm start
./gradlew :ReactAndroid:packageReactNdkLibsForBuck
buck install -r <app>
```